test-suite/businessdayconventions.cpp

```diff
@@ -83,6 +83,7 @@ BOOST_AUTO_TEST_CASE(testConventions) {
         SingleCase(SouthAfrica(), Unadjusted, Date(3,February,2015), Period(1,Months), false, Date(3,March,2015)),
         SingleCase(SouthAfrica(), Unadjusted, Date(3,February,2015), Period(4,Days), false, Date(9,February,2015)),
         SingleCase(SouthAfrica(), Unadjusted, Date(31,January,2015), Period(1,Months), true, Date(28,February,2015)),
+        SingleCase(SouthAfrica(), Unadjusted, Date(30,January,2015), Period(1,Months), true, Date(28,February,2015)),
```

Comment:
May I suggest also adding a test case like 
```
SingleCase(SouthAfrica(), Unadjusted, Date(27,February,2015), Period(1,Months), true, Date(27,March,2015)),
```
which also fails without your change, and might be a bit more clear? The 28 February in the added test case makes it look like an end-of-month adjustment is performed, even though it isn't.  Thanks!